.gitignore

```
@@ -0,0 +1,2 @@
+/.quarto/
+/_site/
```

```suggestion
/_site/

```

We'll add pre-commit to fix these end of file issues